docs/install/README.md

```diff
@@ -454,14 +869,22 @@ Verify that the following CRDs are installed:
 Check that there are two Deployments. The output should look similar to this:
 
 ```console
-kubectl get deployment
+kubectl -n k8ssandra-operator get deployment
 NAME                 READY   UP-TO-DATE   AVAILABLE   AGE
 cass-operator        1/1     1            1           2m
 k8ssandra-operator   1/1     1            1           2m
 ```
 
-### Deploy a K8ssandraCluster
-Now we will deploy a K8ssandraCluster that consists of a 3-node Cassandra cluster and a Stargate node.
+Verify that the `K8SSANDRA_CONTROL_PLANE` environment variable is set to `false`:
+
+```console
+kubectl get deployment k8ssandra-operator-k8ssandra-operator -o jsonpath='{.spec.template.
+spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
+```
+
+#### Deploy a K8ssandraCluster
+Now we will deploy a K8ssandraCluster that consists of a 3-node Cassandra cluster and a 
+Stargate node.
```

Comment:
The below definitions needs to be adjusted so that the object gets created in the `k8ssandra-operator` namespace.